boards/sparc/bm3823/xx3823/scripts/Make.defs

```diff
@@ -55,7 +55,7 @@ ifeq ($(CONFIG_DEBUG_SYMBOLS),y)
 endif
 
 ifneq ($(CONFIG_DEBUG_NOOPT),y)
-  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing -fomit-frame-pointer
+  ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
```

Comment:
Do we need to add `ifeq ($(CONFIG_FRAME_POINTER),y)`?

Reply:
ditto